installation-usage.md

```diff
@@ -576,6 +577,116 @@ NAME                 STATUS      AGE
 my-recording-nginx   Installed   15s
 ```
 
+#### Merging per-container profile instances
+
+By default, each container instance will be recorded into a separate
+profile. This is mostly visible when recording pods managed by a replicating
+controller (Deployment, DaemonSet, etc.). A realistic example might
+be a workload being recorded in a test environment where the recorded
+Deployment consists of several replicas, only one of which is receiving
+the test traffic. After the recording is complete, only the container that
+was receiving the traffic would have container all the syscalls that were
+actually used.
+
+In this case, it might be useful to merge the per-container profiles
+into a single profile. This can be done by setting the `mergeStrategy`
```

Comment:
For my own understanding, but this is useful because it makes the one true profile easier to find, correct?

Otherwise, I could fumble through the profiles and grab one that doesn't contain all the syscalls, since it wasn't used at runtime?

Reply:
That, but also, the recording merging might be more useful than the default no merging in case the profile is recorded in an e2e test suite, where you might test several different test-cases and what you want to allow is probably a union of the cases. Another use-case are replicated pod controllers (daemonSets, deployments) where you run several replicas of the same pod, but the test would only trigger an action on one of them (e.g. you have 10 replicas of nginx listening on port 80, which probably gives you `bind()` and `listen()`, but if you curl from one of them, only that replica would record `accept()`).

You are right that the profile merging is a usability vs security tradeoff. If you wanted to be very secure, you'd review every recorded policy, but since the target "audience" are workloads that are privileged in some way and have no policy at all, even recording a merger of several would probably be quite a bit more secure that none.

You raised some good points, though. Should I add more context to the docs? Please feel free to suggest what might be missing or needs clarification, I might suffer from "tunnel vision" being familiar with the code.